geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionAdvisor.java

```diff
@@ -789,43 +788,49 @@ public synchronized long endOperation(long version) {
     return membershipVersion;
   }
 
-  public void waitForCurrentOperations() {
-    long timeout =
-        1000L * this.getDistributionManager().getSystem().getConfig().getAckWaitThreshold();
-    waitForCurrentOperations(timeout);
-  }
-
   /**
    * wait for the current operations being sent on views prior to the joining of the given member to
    * be placed on communication channels before returning
    *
    * @since GemFire 5.1
    */
-  public void waitForCurrentOperations(long timeout) {
+  public void waitForCurrentOperations() {
+    long timeout =
+        1000L * this.getDistributionManager().getSystem().getConfig().getAckWaitThreshold();
+    waitForCurrentOperations(logger, timeout, timeout * 2l);
+  }
+
+  public void waitForCurrentOperations(Logger alertLogger, long warnMS, long severeAlertMS) {
     // CacheProfile profile = (CacheProfile)getProfile(member);
     // long targetVersion = profile.initialMembershipVersion - 1;
+    long timeout =
+        1000L;// * this.getDistributionManager().getSystem().getConfig().getAckWaitThreshold();
```

Comment:
Looks like a lot of commented out code at the start of this function, probably better to clean this up before pushing.  It also looks like this timeout variable can be removed.

Reply:
Thanks - I'll get rid of that and the "timeout" variable, which isn't used anymore.